EIPS/eip-4511.md

```diff
@@ -0,0 +1,116 @@
+---
+eip: 4511
+title: Execute Past Semantic Abort
+description: Allow clients to choose to continue to execute instructions after EVM bytecode interpretation would revert
+author: Levi Aul (@tsutsu)
+discussions-to: https://ethereum-magicians.org/t/eip-4511-execute-past-semantic-abort/7650
+status: Draft
+type: Standards Track
+category: Core
```

Comment:
I don't think this should be a core EIP as it doesn't affect consensus logic at all. I think either i) the information in the EIP should simply be applied as a PR into an existing client or possible ii) it is created as an `Informational` EIP.